source/slang/slang-stdlib-textures.cpp

```diff
@@ -271,16 +288,16 @@ void TextureTypeInfo::writeGetDimensionFunctions()
                 glsl << "\";\n";
             }
 
-            StringBuilder spirv;
+            auto generateSpirvAsm = [&](StringBuilder& spirv, StringBuilder& imageVar) 
```

Comment:
This lambda can take in a UnownedStringSlice as parameter instead of a StringBuilder.